1.basic/tasks.md

```diff
@@ -3,26 +3,71 @@
 1. create a database named `sports`
 // Answer here ..
 
+use sports
+
 2. list all databases present in local mongod server.
 // Answer here..
 
+admin   0.000GB
+config  0.000GB
+local
+
 3. create 3 collections named `cricket`, `football`, `TT` in sports database.
 
+db.createCollection("cricket")
+{ "ok" : 1 }
+> db.createCollection("football")
+{ "ok" : 1 }
+> db.createCollection("TT")
+
+
 4. add 3 players in each of above collections which should have fields like `name`, `age`, `email`, state and auction_price.
+ db.TT.insert({name:"player1", age:21, email:"abc@gmail.com"})
+
+db.cricket.insert({name:"player2", age:22, email:"abd@gmail.com"})
+
+db.football.insert({name:"player3", age:23, email:"acf@gmail.com"})
+
 
 5. list all collections in sports database.
 
+show collections
+TT
+cricket
+football
+
+
 6. rename `TT` collection to `tennis`.
 
+db.TT.renameCollection("tennis")
+
 7. create a capped collection called `khokho` which should have max 3 documents.
   Try inserting more than 3 and output the result here ?
 
+
+db.createCollection("khokho",{capped:true,max:3});
```

Comment:
It throws this error.
```js
{
	"ok" : 0,
	"errmsg" : "specify size:<n> when capped is true",
	"code" : 14832,
	"codeName" : "Location14832"
}
```
Whenever a capped collection is created, size(in bytes) must be defined along with capped: true boolean.
```js
  db.createCollection("khokho", {capped:true,size: 4096}) // max is optional
```